.github/workflows/quality.yml

```diff
@@ -23,12 +24,56 @@ jobs:
         run: |
           ./gradlew ktlintCheck --continue
       - name: Upload reports
+        if: always()
         uses: actions/upload-artifact@v2
         with:
           name: ktlinting-report
-          path: vector/build/reports/ktlint/*.*
+          path: |
+            */build/reports/ktlint/ktlint*/ktlint*.txt
+      - name: Handle Results
+        if: always()
+        id: get-comment-body
+        run: |
+          results="$(cat */*/build/reports/ktlint/ktlint*/ktlint*.txt */build/reports/ktlint/ktlint*/ktlint*.txt | sed -r "s/\x1B\[([0-9]{1,3}(;[0-9]{1,2})?)?[mGK]//g")"
+          if [ -z  "$results" ]; then
+            body="👍 ✅ 👍"
+          else
+            body="\`👎 ❌ 👎 ${results}\`"
+          fi
+          body="${body//'%'/'%25'}"
```

Comment:
I would apply all the treatment you are doing on the `else` block above and on the `results` var, before affecting the value prefixed with emojis to `body`.

Reply:
Yes I agree, it's really sensitive, if it can support multiple commands inside the else block I will change it

Reply:
I see, thanks